src/restserver.c

```diff
@@ -345,6 +345,17 @@ int main(int argc, char *argv[])
                                &rest_endpoints_cb, &rest);
     ulfius_add_endpoint_by_val(&instance, "GET", "/endpoints", ":name", 10,
                                &rest_endpoints_name_cb, &rest);
+    // Devices
+    ulfius_add_endpoint_by_val(&instance, "GET", "/devices", NULL, 10,
+                               &rest_devices_get_cb, &rest);
+    ulfius_add_endpoint_by_val(&instance, "GET", "/devices", ":id/*", 10,
```

Comment:
You specified all requests to `/device/:id/*`, you should make the request more specific, unless you intend client to make get requests to `/device/:id/hello/world` and such